### PR TITLE
OSDOS-85 Basic builds

### PR DESCRIPTION
**Pull request by bmcelvee**
https://jira.coreos.com/browse/OSDOCS-85

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :snowman:

File: modules/builds-basic-start-source.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+[NOTE]
+====
+Builds triggered from binary input will not preserve the source on the server,
+so rebuilds triggered by base image changes will use the source specified in the
+build configuration.
+====
+
+.Prerequisite
+* Existing build configuration
+
```
Comment (mburke5678) [inline modules/builds-basic-start-source.adoc]: Same

File: modules/builds-basic-start-re-run.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+// * assembly/builds
+
+[id='builds-basic-start-re-run-{context}']
+= Re-running a build
+
+You can manually re-run a build using the `--from-build` flag.
+
+.Prerequisite
+* Existing build configuration
+
```
Comment (mburke5678) [inline modules/builds-basic-start-re-run.adoc]: Same

File: modules/builds-basic-start-logs.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// * assembly/builds
+
+[id='builds-basic-start-logs-{context}']
+= Streaming build logs
+
+You can specify the `--follow` flag to stream the build's logs in stdout.
+
+.Prerequisite
+* Existing build configuration
```
Comment (mburke5678) [inline modules/builds-basic-start-logs.adoc]: Same

File: modules/builds-basic-start-environment-variable.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+// * assembly/builds
+
+[id='builds-basic-start-environment-variable-{context}']
+= Setting environment variables when starting a build
+
+You can specify the `--env` flag to set any desired environment variable for the
+build.
+
+.Prerequisite
+* Existing build configuration
```
Comment (mburke5678) [inline modules/builds-basic-start-environment-variable.adoc]: Same

File: modules/builds-basic-start-build.adoc [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+[id='builds-basic-start-build-{context}']
+= Starting a build
+
+You can manually start a new build from an existing build configuration in your
+current project.
+
+.Prerequisite
+* Existing build configuration
+
```
Comment (mburke5678) [inline modules/builds-basic-start-build.adoc]: I was always taught not to use one-item lists. Maybe a sentence instead? 
_You need an existing build configuration before performing this procedure_ ??

File: modules/builds-basic-delete-buildconfig.adoc [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+.Procedure
+
+To delete a `BuildConfig`, run:
+
+----
+$ oc delete bc <BuildConfigName>
+----
+
+This will also delete all builds that were instantiated from this `BuildConfig`.
```
Comment (mburke5678) [inline modules/builds-basic-delete-buildconfig.adoc]: s/will also delete/also deletes

**Comment by mburke5678** [comment]
@bmcelvee A few nits. Otherwise LGTM

**Pull request merged**